src/anf-dotnetcore-snapshot-policy-sdk-sample/Program.cs

```diff
@@ -123,7 +122,8 @@ private static async Task CreateANFAsync()
                 location,
                 anfAccountName,
                 snapshotPolicyName,
-                newSnapshotPolicy.HourlySchedule);            
+                newSnapshotPolicy.HourlySchedule);
+            //await WaitForAnfResource<SnapshotPolicy>(anfClient, currentSnapshotPolicy.Id);
```

Comment:
>//await WaitForAnfResource<SnapshotPolicy>(anfClient, currentSnapshotPolicy.Id); [](http://example.com/codeflow?start=12&length=80)

Is there a reason why this is commented out?

Reply:
was part of testing and forgot to uncomment it